workspaces/rstudio/Dockerfile

```diff
@@ -75,21 +75,21 @@ RUN apt-get update && \
     # Test:
     gosu nobody true && \
```

Comment:
Interesting, this and the preceding line weren't reformatted 👀

Bug, "this is really hard to do correctly", both?

Reply:
"this is really hard to do correctly, so I don't do it"
https://github.com/reteps/dockerfmt/tree/0.2.8?tab=readme-ov-file#limitations

Reply:
The first line in this `RUN` has the `&& \` at the end still, too. FWIW I think we removed the `gosu nobody true` test from another of the images since it's not strictly needed.

Reply:
Hmm -- I'll have to look into this actually. Probably a bug in the comment parser.

Reply:
Resolved in [f1bf2f7](https://github.com/PrairieLearn/PrairieLearn/pull/11718/commits/f1bf2f7589b65f94da866ddf5dd9739f30377d26).